app/views/forms/show_live.html.erb

```diff
@@ -5,9 +5,50 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l govuk-!-margin-bottom-2">
-      <span class="govuk-caption-l"><%= @form.name %></span><span class="govuk-visually-hidden"> - </span>
-      <%= @form.live? ? t("forms.form_overview.live_title") : t("forms.form_overview.title") %>
+      <%= @form.name %>
     </h1>
-    <p class="govuk-body">Live form placeholder</p>
+    <dl class="govuk-!-margin-bottom-7">
+      <dt class="govuk-visually-hidden">Status</dt>
+      <dd class="govuk-!-margin-0"><%= render FormStatusTagComponent::View.new(status: @form.status) %></dd>
+    </dl>
```

Comment:
I wonder if this should be a component, we are using it in two different places now.

Reply:
I've added an awkwardly named component which wraps the above. It's more a partial than component as it's not very configurable but stops the two diverging.